### PR TITLE
Updated the proposed order calculator to .NET Standard2

### PR DESCRIPTION
**Pull request by navidmatin**
* Updated the project to .NET standard
* Updated tests to .NET core 2.1

### CODE REVIEW

**Comment by mmercurio** [comment]
@navidmatin this looks good!  I was able to successfully build and run all the unit tests under macOS after updating my .NET Core 2.x SDK install. (That's a first!) I believe .NET Core 2.x SDK is also needed to build this on Windows, correct? 

Some comments, but mostly questions: 
1. Have you had a chance to verify the resulting library that is built still works successfully with other projects that depend on it, such as the [LevelUp C# SDK](https://github.com/TheLevelUp/levelup-sdk-csharp)?

2. If there are any new requirements to build and test (e.g., .NET Core SDKs), please update the README with this new information.  

3. How did you convert the .csproj files over to .NET Core Standard?  Did you do this manually, use VS or some other tool?  Whatever the process, it might be nice to document this on the team Confluence page for future projects. 

Please confirm question 1 above and update the README file with any new requirements for building and testing.

**Comment by navidmatin** [comment]
@mmercurio 

> Have you had a chance to verify the resulting library that is built still works successfully with other projects that depend on it, such as the LevelUp C# SDK?
Yes, I tested in one system and it works as expected.

> How did you convert the .csproj files over to .NET Core Standard? Did you do this manually, use VS or some other tool? Whatever the process, it might be nice to document this on the team Confluence page for future projects.
I'm going to document the process but I mostly created a new project and copy pasted files.

**Comment by navidmatin** [comment]
@mmercurio I've made changes to the build server so it now builds.

File: README.md [hunk 1]
```diff
@@ -89,10 +89,12 @@ class Program
...
-  - Visual Studio 2017 (MSBuild 15.0)
+  - Visual Studio 2017 (MSBuild 15.7)
+  - .[NET Core 2.1](https://www.microsoft.com/net/download/dotnet-core/2.1)
+  - .NET Standard 2.0 Support (.NET Framework 4.6.1 and later)
 
 - Release build configuration
-  - Visual Studio 2017 (MSBuild 15.0)
+  - Visual Studio 2017 (MSBuild 15.7)
   - [NuGet CLI](https://docs.microsoft.com/en-us/nuget/tools/nuget-exe-cli-reference)
     - The `nuget` command must be available from the Windows `PATH` variable.
```
Comment (dstiert) [inline README.md]: Should be using the dotnet cli generally. If you're going to call out nuget specifically it'll likely need to be atleast some version to work with the new format.

**Pull request merged**